docs/inspire_crawler.rst

```diff
@@ -0,0 +1,27 @@
+..
+    This file is part of hepcrawl.
+    Copyright (C) 2015, 2016, 2017 CERN.
+
+    hepcrawl is a free software; you can redistribute it and/or modify it
+    under the terms of the Revised BSD License; see LICENSE file for
+    more details.
+
+.. currentmodule:: hepcrawl
+
+
+Inspire-Crawler integration
+===========================
+
+``inspire-crawler`` is used to schedule the crawls in scrapyd. The task
+`schedule_crawl <https://github.com/inspirehep/inspire-crawler/blob/master/inspire_crawler/tasks.py>`_
+is used for this purpose. This task schedules a crawler job using the scrapyd API.
+
+Scrapyd then harvests the record.
+
+A harvested record, after going through the pipeline, is passed to ``inspire-crawler``,
+and more precisely a celery task
+`submit_results <https://github.com/inspirehep/inspire-crawler/blob/master/inspire_crawler/tasks.py>`_
+is scheduled. This task creates the workflow for the record that was harvested, and the record,
+appears in the holdingpen.
```

Comment:
You should probably mention that is the `CrawlResult` the object that is sent with the task, that implements the interaction protocol with inspire-crawler (wrapping the record, any errors and some other metadata).